source/develop/release-management/features/network/autodefine-external-network.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Autodefine External Network
+category: feature
+authors: Ales Musil
+feature_name: Autodefine External Network
+feature_modules: engine
+feature_status: In Development
+---
+
+# Autodefine External Network
+
+## Summary
+
+This feature extends [Provider Physical Network](https://www.ovirt.org/develop/release-management/features/network/provider-physical-network/) 
+adding support for auto-defined external network connected to physical ones. 
+With this feature implemented, when a physical VM network is created, Engine 
+automatically creates an external network based on it, and connects the external
+network to the physical one. This makes it easier to use the former feature, and
+helps oVirt to model host networks and VM networks separately.
+
+Related patches can be found on [gerrit topic:autodefine](https://gerrit.ovirt.org/#/q/topic:autodefine).
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+### Benefit to oVirt
+
+Provider Physical Network is useful for users who would like to 
+manage their network via SDN (such as OVN or Neutron) yet have near-native 
+performance. However, without auto-define feature, such admin would have to 
+create a physical network (and attach it to hosts), create an external network,
+and remember to connect this two networks together. Autodefine helps reducing 
+these administrative tasks. In an OvS cluster with an external provider 
+attached, the external network is automatically created and connected to the 
+physical one.
+
+## Prerequisites
+
+Cluster has to be configured with switch type OvS and valid external provider.
+The user needs to make sure that connection to external provider is valid.
+
+### Limitations
+
+* With OvS switch type in cluster, only external networks are allowed to 
+  be attached to the VM interface via UI. 
+
+* Physical network linked to any external network cannot be removed. This is for
+  keeping the consistency of database. In the future we might add a way to 
+  remove all dependent external networks with physical one. 
+
+* The switch type of existing clusters cannot be changed (from Linux bridge to 
+  OvS or vice versa) via UI. Thus, we currently do not need to worry about 
+  auto-defining external networks on top of existing physical networks.
+
+* Currently updating the physical network won't update auto-defined external 
+  network. This might change in the future. 
```

Comment:
English nits: Currently, updating the physical network will not update the auto-defined external network